pkgs/development/tools/scalingo/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "scalingo";
+  version = "1.28.2";
+
+  src = fetchFromGitHub {
+    owner = pname;
+    repo = "cli";
+    rev = version;
+    sha256 = "sha256-dMiOGPQ2wodVdB43Sk3GfEFYIU/W2K9DG/4hhVxb1fs=";
+  };
+
+  vendorHash = null;
+
+  ldflags = [
+    "-w"
+    "-s"
+  ];
+
+  preConfigure = ''
+    export HOME=$TMPDIR
+  '';
+
+  meta = with lib; {
+    description = "Command Line client for Scalingo PaaS";
+    homepage = "https://doc.scalingo.com/platform/cli/start";
+    changelog = "https://github.com/Scalingo/cli/blob/master/CHANGELOG.md#1282";
```

Comment:
Je pense que vous ne devriez pas faire pointer vers une ligne en particulier, vous devriez enlever la balise `#1282` à la fin de l'url.

Reply:
I removed the tag. Thanks for the feedback!